src/main/webapp/WEB-INF/web.xml

```diff
@@ -45,14 +45,6 @@
         <param-value>/WEB-INF/context/applicationContext.xml</param-value>
     </context-param>
     
-<!--     For some reason Tomcat 7.0.x wasn't resolving the taglibs in the tags folder automatically -->
-    <jsp-config>
-         <taglib>
-             <taglib-uri>http://my.wisc.edu/jsp/jstl/functions</taglib-uri>
-             <taglib-location>/WEB-INF/tags/uwfn.tld</taglib-location>
-         </taglib>
-     </jsp-config>
```

Comment:
Based on your changes when we release a new version of the portlet project (which we should do) we should clean up uPortal's bookmarks overlay.  Please create a uPortal Jira referencing this issue and PR so we can update master.